viewer/packages/cad-model/src/picking/PickingHandler.ts

```diff
@@ -95,11 +95,16 @@ export class PickingHandler {
 
     try {
       for (const cadNode of cadNodes) {
+        //Make current CAD node visible & hide others to fix REV-595
```

Comment:
I think it's a bit unnecessary to mention the issue here in the code :sweat_smile: I expect we won't care about whether or not this was a fix, just that it's correct

Also - I think we usually have a space between the `//` and the comment? :thinking:

Reply:
Better to store original visibility value and restore original value in the finally block.

Reply:
@larsmoa do we allow user to change the visibility of a node? By yesterday discussion, I thought you mentioned, we do not expose `CADNode` as public, I know that user can still make it visible false through `Scene`, just for my knowledge to understand if we expose something to the user to make objects visibility in viewer to false!

Reply:
I found `CogniteCadModel.visible`, so I guess storing previous state make sense